src/Illuminate/Support/Collection.php

```diff
@@ -806,14 +806,16 @@ public function prepend($value, $key = null)
     }
 
     /**
-     * Push an item onto the end of the collection.
+     * Push one or more items onto the end of the collection.
      *
-     * @param  mixed  $value
+     * @param  mixed  $values [optional]
      * @return $this
      */
-    public function push($value)
+    public function push(...$values)
```

Comment:
This breaks the method signature unfortunately.

Reply:
I agree.

Adding another method doesn't make sense for me (e.g. `pushMultiple()`) because I think it makes more sense that it behaves like it's counterpart `array_push()`. So i'll update the PR and hope we get it with the release of Version 7.